xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -63,6 +63,8 @@ struct AppView: View {
 
 //  MARK: Action
 enum AppAction: CustomLogStringConvertible {
+    case recoveryPhrase(RecoveryPhraseAction)
```

Comment:
Wrapper action that boxes the RecoveryPhrase action in order to pass it down to the child component.